main/options/options.cc

```diff
@@ -68,6 +68,7 @@ const vector<PrintOptions> print_options({
     {"autogen-classlist", &Printers::AutogenClasslist, true},
     {"autogen-autoloader", &Printers::AutogenAutoloader, true, false},
     {"autogen-subclasses", &Printers::AutogenSubclasses, true},
+    {"packager", &Printers::Packager},
```

Comment:
Can we call this `package-tree` for consistency with all the other printer options that print `ast::Expression`s?